src/WorldMap.cpp

```diff
@@ -0,0 +1,42 @@
+#include "WorldMap.h"
+#include <iostream>
+#include <algorithm>
+#include <fstream>
+#include <sstream>
+#include <assert.h>
+#include <functional>
+
+WorldMap::WorldMap()
+	: m_tileLayers(),
+	m_currentMapName(),
+	m_tileSheetDetails(32, 40, 54, 0, 0)
+{
+	m_mapTexture.loadFromFile("tilesheet_complete.PNG");
```

Comment:
This should not be hard-coded because the rest of the tile data is coming from a file.